tests/test.js

```diff
@@ -21,29 +22,39 @@ describe("When there are colors with a defined 'on' color", () => {
       plugins: [...tailwindMaterialSurfaces()],
     };
 
-    let componentsCSS = postcss([require("tailwindcss")(config)]).process(
-      "@tailwind components"
-    ).css;
+    let componentsCSS = await postcss([require("tailwindcss")(config)])
```

Comment:
When running tests on this branch, I get the rules in the original order:

```
.surface-a
.interactive-surface-a
.interactive-surface-a:hover 
.interactive-surface-a:active 
.interactive-surface-a:focus-visible 
.interactive-surface-a:disabled 
.dragged-surface-a
```

With the new order, the tests are failing.